microfrontends/app-builder-menu/src/types/api.ts

```diff
@@ -0,0 +1,41 @@
+export interface MenuItem {
+  pbcName: string;
+  bundleName: string;
+  mfeName: string;
+  label: Record<string, string>;
+  target: string;
+  addr: string;
+  org: string;
+}
+
+export interface PbcApiResponse {
+  data: {
+    items: MenuItem[];
+  };
+}
+
+export interface EpcData {
+  'data-organization'?: string; // @TODO Marco tell me final answer after talk with Walter
+  'data-epc-id'?: string; // pbcName
+  'data-submenu'?: string; // @TODO Marco tell me final answer after talk with Nicola
+  'data-menu-item-id'?: string; // we generate this
+}
```

Comment:
@ichalagashvili for this one:
- data-organization: for now add a field `org` on the menu ms API response mocks and use that.
- data-epc-id: is the pbcName on the API response
- submenu: populated only for parent voices that have some children (for example "strapi"). Use pbcName
- data-menu-item-id: please use in this case pbcName + bundleName + label en (lowercase and snakecase)

Reply:
done